src/main/webapp/app/applications/instance/instance.vue

```diff
@@ -19,7 +19,21 @@
                         <a :href="instance.uri" target="_blank">{{ instance.serviceId }}</a>
                     </td>
                     <td class="table-hover">
-                        <span class="badge badge-pill badge-light">{{ instance.instanceId }}</span>
+                        <span class="badge badge-ligth">{{ instance.instanceId }}</span><br/>
+                        <div align="right">
+                            <span class="badge badge-success">{{ instance.metadata.profile }}</span>
+                            <span class="badge badge-primary">{{ instance.metadata.version }}</span>
+                        </div>
+                        <div align="right">
+                            <span v-if="instance.metadata.hasOwnProperty('git-commit') && instance.metadata['git-commit']"
```

Comment:
can we do these checks at the top rather than inside the template? The object chaining is costly and we do it a lot inside the template here.